src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -284,11 +290,27 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
             this.props.onSelectedRegionsChanged([]);
         }
 
+        if ((selectedRegions.find((r) => r.category !== selectedRegions[0].category))
+            === undefined) {
```

Comment:
This looks weird to me, can we do the following?
if (selectedRegions.find((r) => r.category !== selectedRegions[0].category) === undefined)